### ISSUE TITLE
Additional Ticket Status [Feature Request]

### ISSUE THREAD

**Issue by versavius** [opensupports/opensupports#348]
At the moment tickets are either Open or Closed.

We need some new status:

Work In Progress (to indicate a ticket is open but currently being worked on)
On Hold - Awaiting Customer
On Hold - Awaiting Internal
On Hold - Awaiting Third Party

This is needed to help make it clearer as to what the status of a ticket is at a glance, also this allows for reports to be generated from the data.

**Comment by nickbe** [comment]
This can easily be managed with the new tagging system.

**Comment by versavius** [comment]
It can be but these are very common ticket status, i have never seen a ticketing system that does not have an on hold option as part of its list of ticket status.

**Comment by nickbe** [comment]
If we'd have these they would have to be a very easy to handle switch - nothing complicated.

**Comment by jrtderonde** [comment]
True, it would be very nice if administrators could first declare a few different statusses from which other users can choose when assigning tickets. Tags seem to be only visible to the admins and not the users.

**Comment by jrtderonde** [comment]
Tagging system works just great :-)

**Comment by nickbe** [comment]
I just proposed a new status solution #500 for this.
This should solve most wishes just fine.
@ivandiazwm - would you take a look at this 😁
